plutus-core/plutus-core/src/PlutusCore/Builtin/HasConstant.hs

```diff
@@ -38,7 +38,7 @@ fromValueOf uni = fromConstant . someValueOf uni
 {-# INLINE fromValueOf #-}
 
 -- | Wrap a Haskell value (provided its type is in the universe) as a @term@.
-fromValue :: (HasConstant term, UniOf term `Includes` a) => a -> term
+fromValue :: (HasConstant term, UniOf term `Contains` a) => a -> term
```

Comment:
I already forgot we have both `Includes` and `Contains` :scream:

Reply:
Yeah, we might want to discuss how to name those properly, I do agree they are confusing. But that shouldn't block this PR, `Contains` vs `Includes` is an old thing.

To add insult to injury `HasTermLevel` is defined as

```haskell
type HasTermLevel uni = Includes uni
```